hazelcast/src/main/java/com/hazelcast/config/ReplicatedMapConfig.java

```diff
@@ -378,10 +401,13 @@ public void writeData(ObjectDataOutput out) throws IOException {
         out.writeUTF(inMemoryFormat.name());
         out.writeBoolean(asyncFillup);
         out.writeBoolean(statisticsEnabled);
-        out.writeUTF(mergePolicy);
+        if (out.getVersion().isLessThan(Versions.V3_10)) {
```

Comment:
Oh my, this will be missing basically everywhere in the split-brain PRs. I'll try to find all missing ones by the `Versions.V3_10` usages, but not in this PR.